Servers/Infrastructure/OJS.Servers.Infrastructure/Extensions/WebApplicationExtensions.cs

```diff
@@ -95,6 +95,14 @@ public static WebApplication MigrateDatabase<TDbContext>(this WebApplication app
             return app;
         }
 
+        public static WebApplication AddRoles(this WebApplication app)
```

Comment:
`SeedRoles` is a better name IMO, to make clear we add them to the db, not in services or something else.